pkg/sdkserver/sdkserver.go

```diff
@@ -58,14 +58,6 @@ const (
 
 var (
```

Comment:
This can go back to a single line instead of a block now that there is only one variable.